src/stores/Web3Store.js

```diff
@@ -19,25 +22,55 @@ class Web3Store {
     })
   }
 
+  getInfuraLink = (network) => {
+    const infuraToken = 'kEpzZR9fIyO3a8gTqJcI'
+    return `https://${network}.infura.io/${infuraToken}`
```

Comment:
Maybe this should be obtained from an environment variable?

Reply:
Thanks, will add.